<!-- Starts up all nodes for running the robot -->
<launch>
    <!-- Prints output messages to screen for all nodes -->
    <arg name="debug" default="true" />

    <!-- set names of sensors -->
    <arg name="camera" default="camera1" />

    <!-- Load Camera Calibrations -->
    <group ns="camera1/calibration_rgb">
        <rosparam file="$(find cameras)/calibrations/camera1_rgb.yaml" command="load" />
    </group>

    <!-- Load markers -->
    <rosparam file="$(find cameras)/markers/marker_list.yaml" command="load" />

    <!-- Camera Node -->
    <include file="$(find astra_launch)/launch/astra.launch">
        <arg name="camera" value="$(arg camera)" />
    </include>
    <node pkg="cameras" name="marker_testing" type="marker_testing" respawn="true" output="screen">
        <remap from="/camera/rgb/image_raw" to="/$(arg camera)/rgb/image_raw" />
        <param name="camera" value="$(arg camera)" />
    </node>
</launch>
